ubyssey/templates/widgets/frontpage/twitter-front-page.html

```diff
@@ -0,0 +1,5 @@
+<div class="c-twitter-widget-container">
+<a class="twitter-timeline" href="https://twitter.com/ubyssey"></a>
```

Comment:
This shouldn't be hardcoded -- you can return the `{{ handle }}` value that is saved by the widget.